roles/k8s_start_services/tasks/main.yml

```diff
@@ -181,6 +181,14 @@
   when: "'amdgpu-device-plugin' not in k8s_pods.stdout"
   tags: init
 
+- name: Install Spark Operator 
+  command: "helm repo add spark-operator https://googlecloudplatform.github.io/spark-on-k8s-operator"
+  changed_when: true
+  tags: init
+
+- name: Install Spark Operator Namespace
+  command: "helm install my-release spark-operator/spark-operator --namespace spark-operator --create-namespace"
+
```

Comment:
@teiland7 can you add `changed_when: true` and `tags: init` lines to this play?

Please be sure to pull the changes to your local copy first, as I had to resolve some conflicts.

From your terminal, do:
`git pull origin issue-258`

Reply:
@lwilson on my side it has the chaged_when & tags lines. Should I try to pull it again?

Reply:
@teiland7 yes, please do a pull. There were conflicts and when I resolved them it didn't have those 2 lines. Once you fix, you can push back to your repo and it will update here.